<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>590</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttonsLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="buttonOK">
      <property name="text">
       <string>Применить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonCancel">
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>120</y>
     <width>571</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>АППСиРИ</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>551</width>
      <height>195</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="deviceTabLayout">
     <item row="1" column="3" colspan="2">
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>281</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2" rowspan="2">
      <widget class="QGroupBox" name="frameGroupBox">
       <property name="title">
        <string>Кадры</string>
       </property>
       <layout class="QGridLayout" name="_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="frameLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="wordBitsLabel">
            <property name="text">
             <string>Биты данных:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="dataBitsComboBox">
            <property name="toolTip">
             <string>Установка количества бит данных в кадре</string>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="stopBitsLabel">
            <property name="text">
             <string>Стоповые биты:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="stopBitsComboBox">
            <property name="toolTip">
             <string>Установка количества стоповых бит в кадре</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="parityLabel">
            <property name="text">
             <string>Чётность:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QComboBox" name="parityComboBox">
            <property name="toolTip">
             <string>Установка способа проверки на чётность</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="baudRateLabel">
       <property name="text">
        <string>Скорость:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="baudRateComboBox">
       <property name="toolTip">
        <string>Установка скорости (бит/с)</string>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="deviceLabel">
       <property name="text">
        <string>Устройство:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QComboBox" name="flowControlComboBox">
       <property name="toolTip">
        <string>Установка способа управления потоком</string>
       </property>
       <property name="whatsThis">
        <string>Установка способа управления потоком</string>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="flowControlLabel">
       <property name="text">
        <string>Управление потоком:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="deviceComboBox">
       <property name="toolTip">
        <string>Установка имени устройства</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="3" column="4" rowspan="2">
      <spacer name="spacer3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>120</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" colspan="2">
      <spacer name="spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>101</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>350</y>
     <width>571</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>КНУПН</string>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>551</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="deviceTabLayout_">
     <item row="1" column="3" colspan="2">
      <spacer name="spacer1_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>281</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="baudRateComboBox_">
       <property name="toolTip">
        <string>Установка скорости (бит/с)</string>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QComboBox" name="flowControlComboBox_">
       <property name="toolTip">
        <string>Установка способа управления потоком</string>
       </property>
       <property name="whatsThis">
        <string>Установка способа управления потоком</string>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2" rowspan="2">
      <widget class="QGroupBox" name="frameGroupBox_">
       <property name="title">
        <string>Кадры</string>
       </property>
       <layout class="QGridLayout" name="_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="frameLayout_">
          <item row="2" column="0">
           <widget class="QLabel" name="parityLabel_">
            <property name="text">
             <string>Чётность:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QComboBox" name="parityComboBox_">
            <property name="toolTip">
             <string>Установка способа проверки на чётность</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="wordBitsLabel_">
            <property name="text">
             <string>Биты данных:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="dataBitsComboBox_">
            <property name="toolTip">
             <string>Установка количества бит данных в кадре</string>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="stopBitsLabel_">
            <property name="text">
             <string>Стоповые биты:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="stopBitsComboBox_">
            <property name="toolTip">
             <string>Установка количества стоповых бит в кадре</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="4" rowspan="2">
      <spacer name="spacer3_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>120</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="deviceLabel_">
       <property name="text">
        <string>Устройство:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="baudRateLabel_">
       <property name="text">
        <string>Скорость:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="deviceComboBox_">
       <property name="toolTip">
        <string>Установка имени устройства</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <spacer name="spacer2_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>101</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="flowControlLabel_">
       <property name="text">
        <string>Управление потоком:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>30</y>
     <width>571</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Сеть</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="selfPortLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Установка UDP-порта дл приёма ЦУ от АРМ ДУ</string>
       </property>
       <property name="inputMask">
        <string>00000</string>
       </property>
       <property name="text">
        <string>33790</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Сетевой порт:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_3</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOK</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
